client/lib/domains/utils/is-recently-registered.js

```diff
@@ -1,5 +1,8 @@
 import moment from 'moment';
 
-export function isRecentlyRegistered( registrationDate ) {
-	return moment.utc( registrationDate ).isAfter( moment.utc().subtract( 30, 'minutes' ) );
+export function isRecentlyRegistered( registrationDate, minutes = 30 ) {
```

Comment:
Not mandatory at all but we could make that new parameter a bit more explicit, something that gives a hint on its type when you write code that calls that function and autocompletion kicks in:

```suggestion
export function isRecentlyRegistered( registrationDate, numberOfMinutes = 30 ) {
```